src/test/java/teammates/test/cases/logic/TeamEvalResultTest.java

```diff
@@ -491,7 +491,7 @@ private void verifyNormalized(double[] expected, double[] input) {
                 Arrays.toString(TeamEvalResult.normalizeValues(input)));
     }
     
-    private static void showCalculationSteps(int[][] input) {
+    /*private static void showCalculationSteps(int[][] input) {
```

Comment:
I could alternatively take it out and put it to a new class in the `client/` folder. It'll automatically be ignored by the uncommented main rule, and also from test coverage, by the virtue of it being in the `client/` folder. Although it's not a script to connect with the datastore. What do you think?